app/controllers/orders_controller.rb

```diff
@@ -68,6 +68,7 @@ def cancel_order
     if @order.cancelled
       flash[:notice] = 'Order was cancelled successfully'
     else
+      @order.update_inventory(false)
```

Comment:
merge main
then we will not require this change

Reply:
Done